docs/gettingstarted/buildingtest.rst

```diff
@@ -296,11 +296,11 @@ the buildspec will be filtered out if ``--filter maintainers`` is specified. In
 Please see :ref:`buildspec_maintainers` on list of maintainers and breakdown of buildspecs by maintainers.
 
 We can also filter tests by ``type`` field in the buildspec which corresponds to the schema type. In this next example, we filter all tests by script schema type by
-passing option ``--filter type=script``. We inform buildtest to stop after build stage (``--stage=build``) for more details see :ref:`build_stage`.
+passing option ``--filter type=script``.
 
-.. dropdown:: ``buildtest build -b tutorials --filter type=script --stage=build``
+.. dropdown:: ``buildtest build -b tutorials --filter type=script``
 
-    .. command-output:: buildtest build -b tutorials --filter type=script --stage=build
+    .. command-output:: buildtest build -b tutorials --filter type=script
```

Comment:
i actually dont want buildtest to build these tests because `type=script` will end up building alot of tests so please do specify `--dry-run` in both dropdown and command-output

Reply:
👍